src/Templates/NIPSWeb/live.twig

```diff
@@ -23,7 +23,7 @@
             <p>Connecting to server...</p>
             <div id="init-progressbar"></div>
         </div>
-        <div class="ui-dialog-buttonpane ui-widget-content ui-helper-hidden" id="init-overlay-fallback">
+        <div class="ui-dialog-buttonpane ui-widget-content hidden" id="init-overlay-fallback">
```

Comment:
There's a low-priority Issue for that - live is not a production thing and likely doesn't work (hasn't even been renamed for Rapid yet!)